lib/elixir/lib/module.ex

```diff
@@ -1231,6 +1231,40 @@ defmodule Module do
     end
   end
 
+  @doc """
+  Checks if the given attribute has been defined.
+
+  An attribute is defined if it has been registered with `register_attribute/3`
+  or assigned a value. If an attribute has been deleted with `delete_attribute/2`
+  it is no longer considered defined.
+
+  This function can only be used on modules that have not yet been compiled.
+
+  ## Examples
+
+      defmodule MyModule do
+        @value 1
+        Module.register_attribute(__MODULE__, :other_value)
+        Module.put_attribute(__MODULE__, :another_value)
```

Comment:
Is this a get attribute? or is it missing a third argument?

Reply:
missing third value